libc/include/llvm-libc-types/time_t_32.h

```diff
@@ -0,0 +1,14 @@
+//===-- Definition of the type time_t -------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_TYPES_TIME_T_H
```

Comment:
This file and `time_t_64.h` are both using the same header guard as `time_t.h`, `time_t` will not be defined when included from `time_t.h`.

Reply:
Good point – which must mean that nothing at all in the build is _currently_ using this `time_t.h` to redirect to the right sub-header, or else my own test builds would have failed!

Reply:
Did you try with full build mode?  I found out the header guard issue from failed tests when trying your PR in full build mode.

Reply:
Ah, it was in the tests? That explains it.

I'm _mostly_ using full build mode, because I'm testing this libc on bare metal. But I haven't figured out how to run the tests in that setup yet, so that's why I didn't spot this. Good thing you did.

Reply:
Yes, it's good that we can exercise the same code paths on multiple targets.  It does help to find hidden issues.